src/main/features/claim/views/defendant-party-type-selection.njk

```diff
@@ -0,0 +1,44 @@
+{% extends "layout.njk" %}
+{% from "includes/macros/form.njk" import nameInput, csrfProtection, errorSummary, radioOption, saveAndContinueButton %}
+
+{% set title = t("Money claim") %}
+
+{% block content %}
+  <div class="grid-row">
+    <h1 class="heading-xlarge">{{ t("About them") }}</h1>
+    <p>{{ t("Who are you making the claim against?") }}</p>
+    <form method="POST" novalidate>
+      {{ csrfProtection(csrf) }}
+
+      {% set error = form.errorFor('type') %}
+      <div class="form-group {% if error %} form-group-error {% endif %}">
+        <fieldset id="type[label]">
```

Comment:
Same here, possible use for `radioGroup`. Another thing is that it could be extracted as fragment to increase reuse.

Reply:
Reimplemented to use radio group. Further refactoring to be done later.